app/services/external_payment_processor.rb

```diff
@@ -0,0 +1,22 @@
+class ExternalPaymentProcessor
+  require "./lib/stripe_payment_processor.rb"
```

Comment:
It might be overkill, but if I was writing this code I would probably use `#{Rails.root}` instead of `.`. Why leave the current working directory to chance?